### PR TITLE
Add YouTube icon to footer and improve home page UX 

### PR DESCRIPTION
**Pull request by dannyl1u**

### CODE REVIEW

**Comment by dannyl1u** [comment]
lgtm

File: index.html [hunk 1]
```diff
@@ -8,6 +8,7 @@
       name="description"
       content="The Official Website of SFU's Software Systems Student Society (SSSS)"
     />
+    <!-- <meta http-equiv="refresh" content="0; URL='https://www.subway.com/" /> -->
```
Comment (kylesaburao) [inline index.html:11]: wot

File: index.html [hunk 2]
```diff
@@ -195,6 +196,9 @@ <h2 class="h2">Get Our Newsletter</h2>
           <a target="_blank" href="https://www.linkedin.com/company/19037531/">
             <img src="/assets/icons/linked-in-mark.svg" alt="LinkedIn Icon" />
           </a>
+          <a target="_blank" href="https://www.youtube.com/watch?v=DLzxrzFCyOs&ab_channel=AllKindsOfStuff">
+              <img src="/assets/icons/free-youtube-icon-123-thumb.svg" alt="YouTube Icon" style="height: 25px; width: 25px;"/>
```
Comment (kylesaburao) [inline index.html:200]: maybe extract youtube styling since its replicated

Comment (kylesaburao) [reply index.html:200]: to classname / stylesheet

Comment (dannyl1u) [reply index.html]: updated the .svg image itself to the proper size

**Review by kylesaburao — DISMISSED** [review]

**Review by KiaanCastillo — APPROVED** [review]
🚢

File: index.html (hunk 1 above)
Comment (KiaanCastillo) [reply index.html]: long live Sebway

**Pull request merged**